seep-common/src/main/java/uk/ac/imperial/lsds/seep/api/operator/sources/FileConfig.java

```diff
@@ -26,11 +26,19 @@
 	public static final String CHARACTER_SET = "character.set";
 	private static final String CHARACTER_SET_DOC = "Which character set to use for text input. Defaults to Charset.defaultCharset() (JVM default).";
 	
+	public static final String HDFS_SOURCE = "hdfs.source";
```

Comment:
Not sure if we need both the hdfs.source AND the hdfs.uri variables. We could simply assume if hdfs.uri is set that we are talking about an HDFS file.